egs/wsj/s5/steps/data/augment_data_dir.py

```diff
@@ -21,7 +22,7 @@ def GetArgs():
         "babble, for example.  Foreground noises are added sequentially, according to a specified "
         "interval.  See also steps/data/reverberate_data_dir.py "
         "Usage: augment_data_dir.py [options...] <in-data-dir> <out-data-dir> "
-        "E.g., steps/data/augment_data_dir.py --utt-suffix aug --fg-snrs 20:10:5:0 --bg-snrs 20:15:10 "
+        "E.g., steps/data/augment_data_dir.py --utt_prefix aug --fg-snrs 20:10:5:0 --bg-snrs 20:15:10 "
```

Comment:
This seems to be unrelated to the fix you said you were going to make, and anyway it's not compatible-- there are many scripts that use that option, and you are changing its name.
Also, it was a suffix (not a prefix) for a good reason-- making it a prefix would destroy a certain data-being-sorted property that is needed.

Reply:
In my data, utt id is like this: 00001.wav, so I prefer add prefix rather than suffix. I don't want to break the extend name.